ISeriesProvider/LinqToDB.DataProvider.DB2iSeries.csproj

```diff
@@ -58,12 +58,4 @@
 	<ItemGroup>
 	  <PackageReference Include="linq2db" Version="2.3.0" />
   </ItemGroup>
-	<ItemGroup>

```

Comment:
Removed references to dlls in provider project to avoid breaking dependencies when one driver is not installed